packages/fbjs/src/__forks__/Style.js

```diff
@@ -48,15 +48,16 @@ var Style = {
     if (!node) {
       return null;
     }
-    while (node && node !== document.body) {
+    var ownerDocument = node.ownerDocument;
+    while (node && node !== ownerDocument.body) {
       if (_isNodeScrollable(node, 'overflow') ||
           _isNodeScrollable(node, 'overflowY') ||
           _isNodeScrollable(node, 'overflowX')) {
         return node;
       }
       node = node.parentNode;
     }
-    return window;
+    return ownerDocument.defaultView;
```

Comment:
Can we do `ownerDocument.defaultView || ownerDocument.parentWindow` for IE8 compat?